jena-text/src/main/java/examples/JenaTextExample1.java

```diff
@@ -41,9 +41,9 @@
     
     public static void main(String ... argv)
     {
-        Dataset ds = createCode() ;
-        //Dataset ds = createAssembler() ;
-        loadData(ds , "data.ttl") ;
+//        Dataset ds = createCode() ;
+        Dataset ds = createAssembler() ;
```

Comment:
You have changed the existing JenaTextExample1.java that was an example of how to use jena-text/Lucene. Make a  copy instead, leaving the original intact, since we need to have an example of how to configure a Lucene index from code. The new class could be called JenaTextESExample1 or similar.